### PR TITLE
devops/setup-actions

### PR DESCRIPTION
**Pull request by otaviojacobi**
This PR set up the basic automatic testing framework.

The tests in the `test` folder are just sample tests in order to check it's working. Real tests should be added soon.

### CODE REVIEW

**Comment by Theomat** [comment]
Nice job !

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+![main tests status](https://github.com/Theomat/manpac/workflows/Tests/badge.svg?branch=main)
```
Comment (Theomat) [inline README.md:1]: This is the manpac link and not this project link ! Please change !

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -4,12 +4,7 @@ name: Tests
 
 # Controls when the action will run. Triggers the workflow on push or pull request
 # events but only for the master branch
-on:
-  push:
-    branches:
-      - master
-      - dev
-      - setup-actions
+on: [ push ]
```
Comment (Theomat) [inline .github/workflows/test.yml:7]: It's better to specify on which branches this should be run.
Typically these should be master, dev*

**Review by Theomat — CHANGES_REQUESTED** [review]
Nice job, but changes  are needed

File: README.md (hunk 1 above)
Comment (otaviojacobi) [reply README.md]: Oops. Fixed on 1c71a40

File: .github/workflows/test.yml (hunk 2 above)
Comment (otaviojacobi) [reply .github/workflows/test.yml]: Agreed. I changed so it runs on push to `main` and `dev` but it also runs when opening PRs to these branches. This way we know tests are passing before merging :). On 1c71a40

**Review by Theomat — APPROVED** [review]
You fixed everything, and adding the tests on pull_requests is a nice plus !

**Pull request merged**